pkg/api/dbaas/info.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2021 MongoDB.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1 contains API Schema definitions for the dbaas.redhat.com v1alpha1 API group
+// +kubebuilder:object:generate=true
+// +groupName=dbaas.redhat.com
```

Comment:
Is this comment & kubebuilder entry appropriate here? What's it doing?

Reply:
Yes those are required to generate the code with dbaas.redhat.com API group. Without them the generated CRDs would use atlas.mongodb.com (the default one) instead.